intake.py

```diff
@@ -114,6 +114,13 @@ def map_issue(issue, fields):
             if val_mapped:
                 github_issue[field["github"]].append(val_mapped)
 
+        elif field["method"] == "map_append_all":
+            vals_mapped = field["map"].get(knack_field_value)
+
+            if vals_mapped:
+                for val_mapped in vals_mapped:
+                    github_issue[field["github"]].append(val_mapped)
+
```

Comment:
This new case handles the list of GH labels defined in the config that is added in this PR in `config.py`. Previously, the config defines one Knack field value to one GitHub label and was handled by the `map_append` case. The new requirement is to apply multiple GitHub labels based on a single Knack field value so I called it `map_append_all`.